src/librustc_metadata/creader.rs

```diff
@@ -281,53 +347,39 @@ impl<'a> CrateLoader<'a> {
         mut dep_kind: DepKind,
     ) -> Result<(CrateNum, Lrc<cstore::CrateMetadata>), LoadError<'b>> {
         info!("resolving crate `extern crate {} as {}`", name, ident);
+        let mut locate_ctxt = locator::Context {
```

Comment:
why did you move this out? It doesn't seem to be used outside the scope where it used to be in

Reply:
I had to move it out while working on it. I've moved it back now.